res/database/Default/user/sjaele.xml

```diff
@@ -348,21 +348,6 @@
 			<availability year_range_from="" year_range_to="" />
 		</news>
 
-		<news guid="Sjaele-news-Plagiatorin-Forschungsausschuss" type="0" creator="7430" created_by="Sjaele">
```

Comment:
Where did the news go to?

Reply:
to `database_news.xml` where real historical news are now listed chronologically.

Reply:
Odd. Onl viewed it via github/browser and explicitly opened/uncollapsed the common news xml too but ctrl+f did not find it except in the file where it was removed.

Sorry for the fuzz